### PR TITLE
Fixed 31806: New bylines needed for admin settings file upload

### PR DESCRIPTION
**Pull request by playamongthestars**

### CODE REVIEW

File: Modules/Forum/classes/class.ilObjForumAdministrationGUI.php [hunk 1]
```diff
@@ -183,9 +183,16 @@ protected function getSettingsForm()
         $form->addItem($check);
 
         $file_upload = new ilRadioGroupInputGUI($this->lng->txt('file_upload_allowed_fora'), 'file_upload_allowed_fora');
-        $file_upload->addOption(new ilRadioOption($this->lng->txt('file_upload_option_allow'), ilForumProperties::FILE_UPLOAD_GLOBALLY_ALLOWED));
-        $file_upload->addOption(new ilRadioOption($this->lng->txt('file_upload_option_disallow'), ilForumProperties::FILE_UPLOAD_INDIVIDUAL));
-        $file_upload->setInfo($this->lng->txt('file_upload_allowed_fora_desc'));
+        $option_1 = new ilRadioOption($this->lng->txt('file_upload_option_allow'),
+            ilForumProperties::FILE_UPLOAD_GLOBALLY_ALLOWED);
+        $option_1->setInfo($this->lng->txt('file_upload_option_allow_info'));
```
Comment (mjansenDatabay) [inline Modules/Forum/classes/class.ilObjForumAdministrationGUI.php]: Could you please rename this to something like `$globally_allowed_option` ?

File: Modules/Forum/classes/class.ilObjForumAdministrationGUI.php [hunk 2]
```diff
@@ -183,9 +183,16 @@ protected function getSettingsForm()
...
+        $file_upload->addOption($option_1);
+    
+        $option_2 = new ilRadioOption($this->lng->txt('file_upload_option_disallow'),
```
Comment (mjansenDatabay) [inline Modules/Forum/classes/class.ilObjForumAdministrationGUI.php]: Could you please rename this to something like `$per_forum_option` ?

**Review by mjansenDatabay — CHANGES_REQUESTED** [review]
Thx @nmatuschek ,

I suggest to rename the radio option variables.

Best regards,
@mjansenDatabay

File: Modules/Forum/classes/class.ilObjForumAdministrationGUI.php (hunk 2 above)
Comment (playamongthestars) [reply Modules/Forum/classes/class.ilObjForumAdministrationGUI.php]: ok.

**Review by mjansenDatabay — APPROVED** [review]

**Pull request merged**